### ISSUE TITLE
[Feature]*Nice to have* Remember last config

### ISSUE THREAD

**Issue by moraliang** [CoinAlpha/hummingbot#239]
**Feature Suggestion**
// A succinct description of **how you want to achieve the desired behavior**.
When a user re-starts the bot, the bot automatically runs the config s/he uses last time. Only in case that s/he wants to choose or create a new config does s/he need to enter ```config``` before starting. 

**Impact**
// A succinct description of **why you want the _desired behavior_** specified above.
The desired behavior will allow me to start the bot without re-entering ```config``` if I want to use the same config used for the last time. 

**Additional context**
To make the bot more intuitive and save users' time.

**Comment by mifeng** [comment]
Added to backlog: https://app.clubhouse.io/coinalpha/story/4390/start-bot-from-command-line
